flavors/template-Exasol-all-python-3.10-cuda-conda/flavor_base/build_deps/packages/conda_packages

```diff
@@ -1,7 +1,5 @@
 protobuf|4.25.3
 pcre|8.45
 bazel|7.2.1
-gxx_linux-64|12.4.0
-gcc_linux-64|12.4.0
```

Comment:
Why can these dependencies be removed?

Reply:
See https://github.com/exasol/script-languages-release/issues/956#issuecomment-2354802631